homeassistant/components/openweathermap/const.py

```diff
@@ -43,13 +42,12 @@
 ATTR_API_SNOW = "snow"
 ATTR_API_WEATHER_CODE = "weather_code"
 ATTR_API_FORECAST = "forecast"
-ATTR_API_THIS_DAY_FORECAST = "this_day_forecast"
 SENSOR_NAME = "sensor_name"
 SENSOR_UNIT = "sensor_unit"
 SENSOR_DEVICE_CLASS = "sensor_device_class"
 UPDATE_LISTENER = "update_listener"
 COMPONENTS = ["sensor", "weather"]
-FORECAST_MODES = ["hourly", "daily", "freedaily"]
+FORECAST_MODES = ["hourly", "daily", "onecall_hourly", "onecall_daily"]
```

Comment:
`onecall_...` values to seperated consts, e.g. FORECAST_MODE_ONECALL_HOURLY, etc